replication/binlogsyncer.go

```diff
@@ -77,6 +78,10 @@ type BinlogSyncer struct {
 
 	nextPos Position
 
+	useGTID bool
```

Comment:
it's not all right.  if u want sync by GTID, u must have initial GTID set and incremental GTID set information.